lgc/include/lgc/patch/PatchWaveSizeAdjust.h

```diff
@@ -45,6 +45,8 @@ class PatchWaveSizeAdjust final : public llvm::PassInfoMixin<PatchWaveSizeAdjust
   bool runImpl(llvm::Module &module, PipelineState *pipelineState);
 
   static llvm::StringRef name() { return "Patch LLVM for per-shader wave size adjustment"; }
+
+private:
```

Comment:
Why do we need this?

Reply:
this is caused by internal sanitization. will remove it.